WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -28,7 +28,7 @@ class SiteDesignSection: CategorySection {
 class SiteDesignContentCollectionViewController: FilterableCategoriesViewController, UIPopoverPresentationControllerDelegate {
     typealias TemplateGroup = SiteDesignRequest.TemplateGroup
     private let createsSite: Bool
-    private let templateGroups: [TemplateGroup] = [.stable, .singlePage]
+    private let templateGroups: [TemplateGroup] = FeatureFlag.betaSiteDesigns.enabled ? [.stable, .beta] : []
```

Comment:
@enejb I remember from [this older conversation that we decided to use a different approach on iOS and Android](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3125#issuecomment-786346557). I was thinking of aligning the two and:
* pass no groups (defaults to `stable`) when we want the stable designs
* pass `[stable, beta]` when we want to also fetch the beta

Wdyt?

Reply:
The difference between the two was so that we could test the `singe-page` designs on one platform first. Thus we are progressing with aligning the two and test via the `beta` group.